src/components/app-detail/script.js

```diff
@@ -0,0 +1,14 @@
+import { MESSAGE_PLAY_TRACK, MESSAGE_ADD_FAVORITE } from '@/utils/constants';
+
+export default {
+  name: 'app-detail',
+  props: ['trackDetail'],
```

Comment:
As per good practices, it's always recommended to define the type of prop we're expecting

Reply:
got it